include/groonga/table.h

```diff
@@ -208,6 +208,17 @@ grn_table_update(grn_ctx *ctx,
                  unsigned int src_key_size,
                  const void *dest_key,
                  unsigned int dest_key_size);
+/**
+ * \brief Delete all records in table.
+ *
+ * \attention Do not use in multi-threading. Because it might access data that
+ *            has already been deleted and crash when it does.
+ *
```

Comment:
```suggestion
 * \attention Do not use in multi-threading. Because it might access data that
 *            has already been deleted and crash when it does.
 *
 *            You need to reopen the target table and its columns in another processes
 *            when you use this in multi-processing. This removes related files and creates
 *            new files. If you keep using the target table and its columns in another processes,
 *            the target table and its columns in another processes will touch removed data. It
 *            will cause a crash or something wrong. 
```